README.md

```diff
@@ -107,12 +107,10 @@ An optional storage fallback CLI flag `--routing.fallback-targets` can be levera
 ### Storage Caching
 An optional storage caching CLI flag `--routing.cache-targets` can be leveraged to ensure less redundancy and more optimal reading. When enabled, a blob is persisted to each cache target after being successfully dispersed using the keccak256 hash of the existing EigenDA commitment for the fallback target key. This ensure second order keys are succinct. Upon a blob retrieval request, the cached targets are first referenced to read the blob data before referring to EigenDA. 
 
+### Failover Signals
+In the event that the EigenDA disperser or network is down, the proxy will return a 503 status code as a response to POST requests, which rollup batchers can use to failover and start submitting blobs to the L1 chain instead. See https://github.com/ethereum-optimism/specs/issues/434 for more info in the context of the [op stack](#optimism-commitment-mode).
```

Comment:
Let's also add the arb spec: https://hackmd.io/@epociask/SJUyIZlZkx

Reply:
[9e722cf](https://github.com/Layr-Labs/eigenda-proxy/pull/204/commits/9e722cff2ad07b9a0246dc2e34eb8ebb41823cc0)